cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,14 +1,20 @@
-describe('Buy a t-shirt', () => {
+import { MenuContentPage } from "../page/index"
 
-  it('then the t-shirt should be bought', () => {
-    cy.visit('http://automationpractice.com/')
-    cy.get('#block_top_menu > ul > li:nth-child(3) > a').click()
-    cy.get('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click()
-    cy.get('[style*="display: block;"] .button-container > a').click()
-    cy.get('.cart_navigation span').click()
+const menuContentPage = new MenuContentPage()
 
-    cy.get('#email').type('aperdomobo@gmail.com')
-    cy.get('#passwd').type('WorkshopProtractor')
+describe("Buy a t-shirt", () => {
+
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage()
+    menuContentPage.goToTShirtMenu()
+    cy.visit("http://automationpractice.com/")
+    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click()
+    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click()
+    cy.get("[style*=\"display: block;\"] .button-container > a").click();
+    cy.get(".cart_navigation span").click()
+
+    cy.get("#email").type("aperdomobo@gmail.com")
+    cy.get("#passwd").type("WorkshopProtractor")
```

Comment:
Use the page objects that you created to replace these lines by calls to those objects